### PR TITLE
Removed depreciated node modules

### LINKED ISSUE #79: Problem in node modules, using deprecated bcrypt
**Describe the bug**
the most importatnt step to run the development server is to install the dependencies using npm i.
But we are having issues because of the bcrypt module.

**Steps To Reproduce**
fork the repo 
cd client
npm i

**Expected behavior**
Change bcrypt to bcryptjs

Screenshot 2024-06-01 120952

**Desktop (please complete the following information):**
 - OS: Windows
 -  Chrome

### PR DESCRIPTION
**Pull request by Sujal7ss**
## Description

Removed the depreciated bcrypt with non-depreciated  bcryptjs

## How to test
using npm i
now there will be no erros installing the node modules

fixes #79 

(https://help.github.com/articles/closing-issues-using-keywords/)

### CODE REVIEW

File: server/controllers/user.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import bcrypt from "bcrypt";
+import bcrypt from "bcryptjs";
```
Comment (dvjsharma) [inline server/controllers/user.js:1]: I think imports will also need a refactor wherever we are importing `bcrypt`.

**Review by dvjsharma — CHANGES_REQUESTED** [review]
PTAL

File: server/controllers/user.js (hunk 1 above)
Comment (Sujal7ss) [reply server/controllers/user.js:1]: Can you explain more about refractor. I didnt understand your message

Comment (dvjsharma) [reply server/controllers/user.js:1]: What I mean is we might need to change the import statements as well where we are using `bcrypt` to maybe `bcryptjs`. Refer this https://github.com/dvjsharma/Drawn2Shoe/blob/a2b35755ef39450325fbda1e87a46bf180e0612f/server/controllers/user.js#L1

Comment (Sujal7ss) [reply server/controllers/user.js:1]: Oh okay, I already checked all the files. Thats the only one importing bcrypt

Comment (dvjsharma) [reply server/controllers/user.js:1]: After making the adjustment, ensure to thoroughly test the functionality end-to-end. The password hashing process should function properly.

Comment (Sujal7ss) [reply server/controllers/user.js:1]: Only the name is different, all the methods are same in both the modules
Everything is working same as before

Screenshot 2024-06-01 174333

**Review by dvjsharma — APPROVED** [review]
Changes looks good. Tested, working as expected

**Pull request merged**